nucypher/config/storages.py

```diff
@@ -410,7 +422,9 @@ def get(self, checksum_address: str, federated_only: bool, certificate_only: boo
             certificate = self.__read_tls_public_certificate(checksum_address=checksum_address)
             return certificate
         metadata_path = self.__generate_metadata_filepath(checksum_address=checksum_address)
-        node = self.__read_metadata(filepath=metadata_path, federated_only=federated_only)  # TODO: 466
+        node = self.__read_metadata(filepath=metadata_path,
+                                    blockchain=self.blockchain,
```

Comment:
There is an emerging pattern of passing both `federated_only` and `blockchain` - Can we use merely the presence of `blockchain` to indicate decentralized / federated operating modes?

Reply:
Yes, we should try to avoid by design incompatible parameters like those. Isn't it better if we do that change in Earl Gray?

Reply:
https://github.com/nucypher/nucypher/issues/1236